### DISCUSSION TITLE
Running batch skill / collection import from the jar - fails (access denied)

### DISCUSSION THREAD

**Discussion by jotepen** [wgu-opensource/osmt#102] [Asking for Help] [answered]
Hi,

I'm attempting to import the skills from the provided CVS using the command below, but it keeps failing with Access denied.

```java -jar -Dspring.profiles.active=dev,import api/target/osmt-api-<version>.jar --csv=path/to/csv --import-type=batchskill  ```

Error message:

```
Unable to obtain connection from database (jdbc:mysql://osmt_db_user:password@0.0.0.0:3306/osmt_db) for user 'osmt_db_user': Access denied for user 'osmt_db_user'@'172.31.26.34' (using password: YES)
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
SQL State  : 28000
Error Code : 1045
Message    : Access denied for user 'osmt_db_user'@'172.31.26.34' (using password: YES)
```
Where is this command pulling this data from?  After doing some research, it looks like the application.properties file contains the data for the jbdc command, but when I made multiple changes in this file, I never saw any changes occur in the jbdc command.

Thanks.

**Comment by jotepen** [comment, marked as answer]
I rebuilt the OSMT server and now this is no longer an issue.

**Comment by JohnKallies** [comment]
I'm glad it's not a problem any longer.

I'm still learning about the organizations that use OSMT, and how people interact with it. How are you using the CSV file for import? Did you export from another instance? Build a file in Excel?
